test/e2e/controller_suite_test.go

```diff
@@ -0,0 +1,72 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/operator-framework/combo/api/v1alpha1"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	kscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	kubeclient client.Client
+)
+
+func TestE2E(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	SetDefaultEventuallyTimeout(1 * time.Minute)
+	SetDefaultEventuallyPollingInterval(1 * time.Second)
+
+	RunSpecs(t, "E2E Suite")
+}
+
+var _ = BeforeSuite(func() {
```

Comment:
Is there any reason we'd need an AfterSuite in the current test setup? It doesn't looks like we do any real test setup in the BeforeSuite (e.g. create a kind cluster like in OLM/registry), so I think we're all set, but wanted to make a note in any case.

Reply:
I'm not coming up with anything as I think about that would need to happen in an AfterSuite call. The BeforeSuite here is mainly just making sure that everything is defined that the tests are assuming exist which you were pointing out. So nothing really to clean up unless we'd like the E2E suite create its own Kind cluster.